chart/kubedb-catalog/templates/postgres.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: kubedb.com/v1alpha1
+kind: PostgresVersion
+metadata:
+  name: "9.6"
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: "{{ template "kubedb-catalog.name" . }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+spec:
+  db:
+    image: "kubedb/postgres:9.6"
```

Comment:
I think there should be a values.yaml with an option to specify registry . Example: https://github.com/kubedb/cli/blob/master/chart/kubedb/values.yaml#L6

This will allow someone using a private registry to update the image properly.